biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -515,6 +440,8 @@ subroutine spawn_patches( currentSite, bc_in)
     ! If nocomp is not enabled, then this is not much of a loop, it only passes through once.
     nocomp_pft_loop: do i_nocomp_pft = min_nocomp_pft,max_nocomp_pft
 
+    disturbance_type_loop: do i_disturbance_type = 1,N_DIST_TYPES
+
     ! calculate area of disturbed land, in this timestep, by summing contributions from each existing patch. 
```

Comment:
lets get an indent in here, like the loop label

Reply:
OK I did a big auto-indent for this subroutine.